### PR TITLE
Feature/117 - Ability to Browse Companies PR

### PR DESCRIPTION
**Pull request by kocsenc**

### CODE REVIEW

File: client/app/views/feed.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+          <dir-pagination-controls
+            template-url="/views/templates/dirPagination.tpl.html"
+            boundary-links="false"></dir-pagination-controls>
+          <!-- https://github.com/michaelbromley/angularUtils/tree/master/src/directives/pagination#installation -->
+        </div>
+
+      </div>
+
+
+      <!-- MENU ATTEMPT -->
```
Comment (jcdesimp) [inline client/app/views/feed.html]: What's all this commented out stuff?

**Comment by jcdesimp** [comment]
Seem all good to me, what's that commented out HTML I pointed out though?

**Comment by kocsenc** [comment]
@jcdesimp , It's for reference of the library documentation for whoever has to manage the pagination.

**Comment by jcdesimp** [comment]
@kocsenc oh i put the reply in a bad spot but i meant the like 20 lines of comments underneath the line I left the comment on.

**Pull request merged**